modules/end-user-guide/partials/assembly_workspaces-overview.adoc

```diff
@@ -7,12 +7,12 @@
 
 :context: developer-workspaces
 
-{prod} provides developer workspaces with everything needed to a code, build, test, run, and debug applications. To allow that, the developer workspaces provide four main components:
+{prod} provides developer workspaces with everything you need to code, build, test, run, and debug applications:
 
-. The source code of a project.
-. A web-based integrated development environment (IDE).
-. Tool dependencies, needed by developers to work on a project.
-. Application runtime: a replica of the environment where the application runs in production.
+. Your project's source code
```

Comment:
These all need to be put in a bullet point list (using `*` instead of `.`) with the dot at the end.

Reply:
:) Thanks! Periods made sense in the previous version, but asterisks make more sense in the updated version. Changed.
About the missing dots at the end, those are missing on purpose: IBM Style Guide: p. 89 "Write lists so that all or none of the items start with complete sentences. ... If list items comprise only sentence fragments, do not include any end punctuation."